### PR TITLE
Search: Sort by updated at and created at

### PR DESCRIPTION
**Pull request by FZambia**
**What this PR does / why we need it**:

Extends Enterprise with sorting by dashboard/folder `updated_at` field. Seems useful to me, to discover dashboards with recent activity, or stale dashboards.

UPD. And by `created_at`

### CODE REVIEW

**Comment by ArturWierzbicki** [comment]
I like the idea, but something is off with the implementation, notice how the filter changes to `Updated (most)` after I select `Updated time`.

https://user-images.githubusercontent.com/23451458/173560402-dfb51f91-269e-4dfd-808b-b022d8c7bf98.mp4


Unrelated, but I don't love the dropdown component for sort selection. It wasn't obvious to me that I can scroll inside :D

**Comment by FZambia** [comment]
> Unrelated, but I don't love the dropdown component for sort selection. It wasn't obvious to me that I can scroll inside :D

+1, that was unobvious for me also

File: pkg/services/searchV2/bluge.go [hunk 1]
```diff
@@ -33,6 +33,7 @@ const (
 	documentFieldTransformer = "transformer"
 	documentFieldDSUID       = "ds_uid"
 	documentFieldDSType      = "ds_type"
+	documentFieldUpdatedAt   = "updated_at"
```
Comment (ryantxu) [inline pkg/services/searchV2/bluge.go:36]: can we also add `created_at`?

**Comment by ryantxu** [comment]
This looks good! we should also add `created_at` index.  

In the table view, we will also need something to format the times reasonably... I can take a look at that tomorrow.

File: pkg/services/searchV2/bluge.go (hunk 1 above)
Comment (FZambia) [reply pkg/services/searchV2/bluge.go:37]: added

**Comment by ryantxu** [comment]
🔥 

image

**Review by ryantxu — APPROVED** [review]

**Pull request merged**